### PR TITLE
Various DB fixe to greatly improve the game world

### PR DESCRIPTION
**Pull request by Daribon**
Wrote this sometime back in 2017.

DB update to fix dwarf mages crashing the server, the half invisible NPCs caused by incorrect unitflags and most importantly, removes all gameobject and creatures with displayids not found within the client.
By removing all blue boxes, the game world seems much more playable.

### CODE REVIEW

**Comment by loonbg** [comment]
Nice to seen some one work on this..  i has hope some one will take emulator and add some feature like finish spells / walking animation on npc .. but sadly no one in interest of this pice of wow history

**Comment by barncastle** [comment]
@Daribon Apologies I've only just noticed this, will merge - thanks!
@loonbg I've been debating rewriting this properly (now that I know what I'm doing-ish), however the problem is the lack of database and formulas - and time/commitment. To break it down:
- I have an idea on how to get a near-enough complete 0.6.0 item database but its not quick or easy to do. This also creates the problem of unbalanced combat due to 0.6.0 item stats vs 1.12.1 creature stats
- Most 1.12.1 formulas can't be used as some stats are either missing or worked differently, plus stat distribution is completely different - again - unbalancing combat
- The rest of the tables will need to be reversed from 1.12.1 meaning manually fixing entries - of which there are millions (there is potential for some automated guessing/correction though)
- The biggest problem of all is the relation between all entities and DBCs e.g. one missing DB entry could break multiple quests, creatures, gameobjects and npcs which in turn breaks even more things and so forth...

My only thought on the above is approaching WoWHead and seeing if they'd be willing to share their oldest backups of thottbot and allakhazam - which is hugely doubtful and may not even be usable. Alternatively; to bin off the data completely and create a functional sandbox - but even that is a huge undertaking.

Tl;dr - its an unfathomable amount of work that probably isn't justifiable.

**Comment by Daribon** [comment]
I do have tons of old DBs and emulators for pre 1.12. However those DBs wouldn't really help out in terms of anything creature related.

**Comment by loonbg** [comment]
well there is some database (they are mess as spawns but still something.. ) from period of wow beta 0.7/0.6  (examle: wowpython)
